.gitignore

```
@@ -1,12 +1,7 @@
-build
-dist
-_build
 docs/man/*.gz
 docs/source/api/generated
 docs/source/config.rst
 docs/gh-pages
```

```suggestion
docs/gh-pages
nbclassic/i18n/*/LC_MESSAGES/nbjs.json
```

When I `pip install -e .` from this PR, I see that the `nbclassic/i18n/*/LC_MESSAGES/nbjs.json` files now get picked up by git as untracked changes. However since these files will be explicitly declared as required in #339, I'm wondering if we can leave this pattern in the .gitignore and rely on the pyproject.toml to ensure the files are included in the `wheel` and `sdist`?

---

Thanks, good point @RRosio!